### DISCUSSION TITLE
What doese "BE_COP" at ALTHERMA(HPSU6_ULTRA) file?

### DISCUSSION THREAD

**Discussion by ap20132022** [raomin/ESPAltherma#336] [Q&A]
Hi everyone,
What doese "BE_COP" at ALTHERMA(HPSU6_ULTRA) file?
I use it but it always shows 0.
Thanks for your answer.

**Comment by camielverdult** [comment]
It's also always 0 for EHVH08S23EJ9W + ERGA08EAV3H7
